### PR TITLE
feat: theme toggler

### PR DESCRIPTION
**Pull request by cindy-sf**
https://user-images.githubusercontent.com/38938357/126867012-559acd07-4aa8-46bb-b9b4-f1de87cae575.mp4

### CODE REVIEW

File: src/styles/theme.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export type ThemeModeType = 'dark' | 'light'
 export interface ThemeAttributes {
   mode: ThemeModeType
   BACKGROUND: string
+  BOTTOM_SHEET_BACKGROUD: string
```
Comment (j0nathansf) [inline src/styles/theme.ts]: What was it really necessary to add another attribute?

**Review by j0nathansf — DISMISSED** [review]
It seems good to me

File: src/styles/theme.ts (hunk 1 above)
Comment (cindy-sf) [reply src/styles/theme.ts]: Because isn't the same color than the main background

Comment (cindy-sf) [reply src/styles/theme.ts]: And I do a spelling mistake to ^^

Comment (j0nathansf) [reply src/styles/theme.ts]: Ok

**Review by j0nathansf — APPROVED** [review]

**Pull request merged**